spec/index.html

```diff
@@ -612,6 +624,49 @@ <h3>Version Labels</h3>
         </tr>
       </tbody>
     </table>
+
+    <p>For serializations supporting in-line version annouoncement,
+      the version announcement SHOULD be made early in the document
+      and certainly before serializing any feature depending on that version.</p>
+
+    <section id="server-considerations" class="informative">
+      <h4>Server considerations</h4>
+      <p>When serializing a graph or dataset which uses features incompatible
+        with a requested version, servers can consider different alternatives:</p>
+
+      <ol>
+        <li>Downgrade <a>Triple Terms</a> by using an algorithm such
+          as <em>Unstar</em> ("1.2" to "1.2-basic" or "1.1").
+          <div class="ednote">Fix this when such an algorithmm is defined.</div></li>
+        <li>Downgrade <a>directional language-tagged strings</a>
+          to a <a>literal</a> with a <a>datatype IRI</a>,
+          using <a data-cite="JSON-LD11#the-i18n-namespace">the `i18n` namespace</a>, as defined in [[JSON-LD11]]
```

Comment:
Minor edit suggestion:
```suggestion
        <li>Replace <a>directional language-tagged strings</a>
          by <a>literals</a> with a <a>datatype IRI</a> that
          uses the <a data-cite="JSON-LD11#the-i18n-namespace">`i18n` namespace</a>, as defined in [[JSON-LD11]]
```